source/Scrapers/TeamCityBuildScraper.cs

```diff
@@ -32,27 +32,27 @@ protected override void Scrape()
             teamCityClient.ConnectWithAccessToken(teamCityToken);
 
             var hungBuilds = teamCityClient.Builds
-                .GetFields("count,build(id,probablyHanging,buildTypeId)")
-                .ByBuildLocator(BuildLocator.WithDimensions(running: true), new List<string> { "hanging:true" })
+                .GetFields("count,build(id,probablyHanging,buildTypeId,composite)")
+                .ByBuildLocator(BuildLocator.WithDimensions(running: true), new List<string> { "hanging:true", "composite:false" })
                 .ToArray();
 
-            var gauge = metricFactory.CreateGauge("probably_hanging_builds", "Count of running builds that appear to be hung", "buildTypeId");
+            var gauge = metricFactory.CreateGauge("probably_hanging_builds", "Running builds that appear to be hung", labelNames: new []{"buildTypeId", "buildId"});
```

Comment:
🤔 Do we still need to keep the buildTypeId as a dimension? What if we just reported on specific hung builds?

Reply:
I left this in so that we can spot trends for common buildTypes that hang